teammapper-backend/src/map/services/maps.service.ts

```diff
@@ -20,7 +20,7 @@ import MalformedUUIDError from './uuid.error'
 
 @Injectable()
 export class MapsService {
-  private readonly logger = new Logger(MapsService.name)
+  public readonly logger = new Logger(MapsService.name)
```

Comment:
why does need to be public?

Reply:
It's an additional step during the newly added test cases to check whether the logger was triggered (as it should output a warning if the `addNode` Promise was rejected). For that the logger needs to be accessible. I can change it to supply it through the constructor instead and use a mock logger in the tests, but that will require some additional work.

Reply:
I think this is already dependency injected: https://docs.nestjs.com/techniques/logger
Changing the visibility just for a test case does not sound right. It should already be possible to provide the logger in this test case as a dependency. For instance (haven't tested): https://stackoverflow.com/questions/68394971/how-to-mock-nestjs-built-in-logger-in-jest

Reply:
Mocking it in the test fixture did nothing, but it's possible to spy on `Logger.prototype` and get the warning that way, without setting the logger to public.